Source/DafnyLanguageServer/Language/DafnyPluginsOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.Dafny.LanguageServer.Language {
+  /// <summary>
+  /// Options for the resolution and compilation pipeline
+  /// </summary>
+  public class DafnyPluginsOptions {
+    /// <summary>
+    /// The IConfiguration section of the compiler options.
+    /// </summary>
+    public const string Section = "Dafny";
```

Comment:
Maybe to stick to the naming scheme of the other configuration sections. You may consider changing this section's name to `plugins` and the property/option to `list`.

Reply:
Other configuration sections do not have options that are lists. I don't think it makes sense to name a field based on its type. Moreover, these are plugins for Dafny core itself, not the DafnyLanguageServer, so the option" --dafny:plugins:0=..." makes more sense than "--plugins:list:0=...". Can you please elaborate?

Reply:
I agree that it's not good to name fields/properties by their type. This one's indeed a hard one. My suggestion was based on the naming scheme that targets a certain feature of Dafny, e.g., `verifier` or `ghost`. That's why I came up with the suggestion of naming the section `plugins` to combine settings related to the plugin feature. Unfortunately, I couldn't come up with a better name than `--plugins:list` in the meantime. But as I noted, I don't like this name either; thus, I'm fine with `--dafny:plugins`, but you may have another idea with my motivation.